lib/checkleakautovar.cpp

```diff
@@ -384,6 +396,17 @@ void CheckLeakAutoVar::checkScope(const Token * const startToken,
                             // Fixme: warn about leak
                             alloctype.erase(innerTok->varId());
                         }
+                        const Library::AllocFunc* g = mSettings->library.realloc(innerTok->tokAt(2));
+                        if (g && g->arg == -1) {
```

Comment:
Please break out this copy/pasted block of code into a method.

Reply:
Done.